sdk/core/Azure.Core.Experimental/api/Azure.Core.Experimental.netstandard2.0.cs

```diff
@@ -24,6 +24,52 @@ namespace Azure.Core
         public override string ToString() { throw null; }
         public string ToString(System.Text.Encoding encoding) { throw null; }
     }
+    public partial class DynamicJson : System.Dynamic.IDynamicMetaObjectProvider
+    {
+        public DynamicJson(string json) { }
+        public DynamicJson(System.Text.Json.JsonElement element) { }
+        public Azure.Core.DynamicJson this[int index] { get { throw null; } set { } }
+        public Azure.Core.DynamicJson this[string property] { get { throw null; } set { } }
+        public static Azure.Core.DynamicJson Array() { throw null; }
+        public static Azure.Core.DynamicJson Array(params Azure.Core.DynamicJson[] values) { throw null; }
+        public static Azure.Core.DynamicJson Array(System.Collections.Generic.IEnumerable<Azure.Core.DynamicJson> values) { throw null; }
+        public static Azure.Core.DynamicJson Create(System.Text.Json.JsonElement element) { throw null; }
+        public System.Threading.Tasks.Task<T> DeserializeAsync<T>(Azure.Core.JsonObjectSerializer serializer) { throw null; }
+        public T Deserialize<T>(Azure.Core.JsonObjectSerializer serializer) { throw null; }
+        public T Deserialize<T>(System.Text.Json.JsonSerializerOptions? options = null) { throw null; }
+        public System.Collections.Generic.IEnumerable<Azure.Core.DynamicJson> EnumerateArray() { throw null; }
+        public System.Collections.Generic.IEnumerable<System.Collections.Generic.KeyValuePair<string, Azure.Core.DynamicJson>> EnumerateObject() { throw null; }
+        public int GetArrayLength() { throw null; }
+        public bool GetBoolean() { throw null; }
```

Comment:
I was purposefully trying to mirror JsonElement: https://docs.microsoft.com/en-us/dotnet/api/system.text.json.jsonelement?view=netcore-3.1.

Reply:
Yeah, I wasn't really expecting that.  When I see this API, I wonder why we don't just add a couple of extension methods to `JsonElement` like `Serialize<T>`, `AsDynamic`, etc.  I was thinking this type would be much smaller and focused exclusively on objects to avoid all these other accessors.

Reply:
This type can't be focused just on objects, services like digitaltwins  support taking arbitrary JSON so the root can be an object or an array maybe even a primitive.

Reply:
And maybe we're trying to solve too much here?  Maybe we should tell folks if you want to use primitives or arrays with DigitalTwins that you need to work with raw JSON elements.  I don't think I'd want to use this type as it exists right now for Search because it's way too general and I think would confuse what's already a tricky part of the Search API.